client/src/test/scala/cakesolutions/kafka/ConfigureSerializationSpec.scala

```diff
@@ -46,7 +46,8 @@ class ConfigureSerializationSpec extends KafkaIntSpec{
          """.stripMargin
       ), keySerializer, valueSerializer)
 
-    val _ = KafkaProducer(conf)
```

Comment:
Producer thread will stay in JVM even after finishing the test.

Reply:
Here is the picture I recorded from VisualVM.
![screen shot 2018-04-19 at 17 16 43](https://user-images.githubusercontent.com/5125598/39000626-0c9fb3b4-4427-11e8-9013-1b5505639d1b.png)